core/rebin.cpp

```diff
@@ -123,7 +126,7 @@ Variable rebin(const VariableConstProxy &var, const Dim dim,
   // Rebin could also implemented for count-densities. However, it may be better
   // to avoid this since it increases complexity. Instead, densities could
   // always be computed on-the-fly for visualization, if required.
-  expect::unit(var, units::counts);
+  // expect::unit(var, units::counts);
```

Comment:
@SimonHeybrock Not sure what to do with this check, except to remove it. Any suggestions?

Reply:
Maybe it makes sense to allow dimensionless in addition to counts, but forbid anything else?